src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,26 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  byQuery: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, byQuery }) => {
+  const visibleMovies = movies.filter((movie) => {
+    const lowerTitle = movie.title.toLowerCase();
+    const lowerDescription = movie.description.toLowerCase();
+    const lowerByQuery = byQuery.toLowerCase();
```

Comment:
Will be better to move this code out of filter method
Now you do the same functionality for each callback call